### PR TITLE
changes related to libreport's reporter-systemd-journal

### PR DESCRIPTION
**Pull request by mhabrnal**
Related documentation pages:
https://github.com/abrt/libreport/wiki/Reporter-systemd-journal
https://github.com/abrt/abrt/wiki/systemd-journal-catalog-messages

### CODE REVIEW

**Comment by mhabrnal** [comment]
retest this please

File: src/plugins/abrt-action-ureport [hunk 1]
```diff
@@ -177,7 +171,8 @@ if __name__ == "__main__":
         if exitcode == 70 and watch.lower() in ["yes", "on", "1"]:
             if reported_to and reported_to != "" and get_bugzilla_reports(reported_to):
                 log(_("Adding you to CC List of the existing bugzilla bug"))
-                run_event("watch_Bugzilla", dirname)
+                if not run_event_on_problem_dir(dirname, "watch_Bugzilla"):
+                    log1("Didn't find definition of event 'watch_Bugzilla'")
```
Comment (jfilak) [inline src/plugins/abrt-action-ureport]: This is not always correct error message.

File: src/cli-ng/abrtcli/cli.py [hunk 2]
```diff
@@ -246,10 +245,10 @@ def retrace(args):
 
         if remote:
             print(_('Remote retracing'))
-            run_event('analyze_RetraceServer', prob)
+            libreport.run_event_on_problem_dir(prob.path, 'analyze_RetraceServer')
```
Comment (jfilak) [inline src/cli-ng/abrtcli/cli.py]: `run_event` was always logging an error message but `run_event_on_problem_dir` logs errors only if you pass True as the 3rd argument.

**Comment by jfilak** [comment]
f22e33e testsuite: add test for reporter-systemd-journal
537f448 a-a-analyze-python: create exception_type element
dbea230 a-a-analyze-xorg: create crash_function into dump dir
f81b3d4 koops: create crash_function element
cf3f037 a-a-analyze-python: create crash_function element
c158e69 a-a-analyze-c: create crash_function element
1520f73 spec: add formatting files for reporter-systemd-journal
2ab9a73 reporter-systemd-journal: add formatting files
62704f1 vmcore: /var/tmp/abrt is no longer a dump location
ba0620d events: add event report_systemd-journal to all addons
399c0e8 abrt-action-notify: notify to systemd journal
5cb0f0a spec: add abrt's catalog source files
ed18912 journal-catalog: add abrt's catalog source files

Well done! Fix the issue we have found please :)

**Pull request closed without merging**